app/services/bulk_upload/sales/year2024/row_parser.rb

```diff
@@ -1443,7 +1443,7 @@ def validate_incomplete_soft_validations
 
         field_mapping_for_errors[interruption_screen_question_id.to_sym]&.each do |field|
           if errors.none? { |e| e.options[:category] == :soft_validation && field_mapping_for_errors[interruption_screen_question_id.to_sym].include?(e.attribute) }
-            error_message = [display_title_text(question.page.title_text, log), display_informative_text(question.page.informative_text, log)].reject(&:empty?).join(". ")
+            error_message = [display_title_text(question.page.title_text, log), display_informative_text(question.page.informative_text, log)].reject(&:empty?).join(" ")
```

Comment:
Instead of joining error message test with a full stop, we now include the full stop in the text set in config/locales/en.yml. Keeping this consistent with other error messages.